lib/audit_help.c

```diff
@@ -68,6 +74,44 @@ void audit_logger (int type, MAYBE_UNUSED const char *pgname, const char *op,
 	}
 }
 
+/*
+ * This function will log a message to the audit system using a predefined
+ * message format. For additional information on the group account lifecycle
+ * events check
+ * <https://github.com/linux-audit/audit-documentation/wiki/SPEC-User-Account-Lifecycle-Events>
+ *
+ * Parameter usage is as follows:
+ *
+ * type - type of message.A list of possible values is available in
+ *        "audit-records.h" file.
+ * op  -  operation. "adding user", "changing finger info", "deleting group"
```

Comment:
The strings you're actually passing don't have spaces.  Please revise this documentation.

Reply:
Yeah, and `changing finger info` doesn't even exist. Updated to provide real-world examples.